collectors/hra/collector.py

```diff
@@ -0,0 +1,137 @@
+                data['contact_email'] = application['ContactEmail']
+                data['application_title'] = application['ApplicationTitle']
+                data['study_type_id'] = application['StudyTypeID']
+                data['study_type'] = application['StudyType']
+                data['sponsor_org'] = application['SponsorOrganisation']
+                data['research_programme'] = application['ResearchProgramme']
+                data['data_coll_arrangements'] = application['DataCollectionArrangements']
+                data['establishment_org'] = application['EstablishmentOrganisation']
+                data['establishment_org_address_1'] = ['EstablishmentOrganisationAddress1']
+                data['establishment_org_address_2'] = application['EstablishmentOrganisationAddress2']
+                data['establishment_org_address_3'] = application['EstablishmentOrganisationAddress3']
+                data['establishment_org_post_code'] = application['EstablishmentOrganisationPostcode']
+                data['decision'] = application['Decision']
+                data['decision_date'] = application['DecisionDate']
+                data['human_tissue_license'] = application['HumanTissueAuthorityStorageLicence']
+                data['rtb_title'] = application['RTBTitle']
+                data['research_database_title'] = application['ResearchDatabaseTitle']
+                data['application_full_title'] = application['ApplicationFullTitle']
+                data['isrctn'] = application['ISRCTN']
+                data['nct'] = application['NCT']
+                data['additional_ref_numbers'] = application['AdditionalReferenceNumbers']
+                data['duration_of_study_in_uk'] = application['DurationOfStudyInUK']
+                data['research_summary'] = application['ResearchSummary']
+                data['eudra_ct'] = application['EudraCT']
+                data['social_value'] = application['SocialValue']
+                data['recuitment_arrangements'] = application['RecruitmentArrangements']
+                data['risk_and_benefit'] = application['RiskAndBenefit']
+                data['participants_protection_and_care'] = application['ParticipantsProtectionAndCare']
+                data['informed_consent'] = application['InformedConsent']
+                data['applicant_and_staff_suitability'] = application['ApplicantAndStaffSuitability']
+                data['independent_review'] = application['IndependentReview']
+                data['supporting_info_suitability'] = application['SupportingInfoSuitability']
+                data['other_comments'] = application['OtherComments']
+                data['research_summary_suitability'] = application['ResearchSummarySuitability']
+
+                # Insert into database only if app_id not inserted already
+                check_id = conn['warehouse'].query("select count(*) from hra where application_id='%s'" % str(application['ApplicationID']))
```

Comment:
I mean something like this instead of string interpolation:
```
conn['warehouse'].query('select count(*) from hra where application_id=?', str(application['ApplicationID']))
```
http://docs.sqlalchemy.org/en/rel_0_9/core/connections.html#sqlalchemy.engine.Connection.execute